src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
+      <div class="calendar__day-1"></div>
+      <div class="calendar__day-2"></div>
```

Comment:
no needs to use different selectors
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
isn't fixed